src/rules-engine/Property.js

```diff
@@ -62,15 +62,24 @@ class Property extends EventEmitter {
    * @return {Promise} resolves to property's value
    */
   async get() {
-    return await Things.getThingProperty(this.thing, this.id);
+    try {
+      return await Things.getThingProperty(this.thing, this.id);
+    } catch (e) {
+      console.warn('Rule get failed', e);
+    }
   }
 
   /**
    * @param {any} value
-   * @return {Promise} resolves if property is set to value
+   * @return {Promise} resolves when set is done
    */
-  async set(value) {
-    await Things.setThingProperty(this.thing, this.id, value);
+  set(value) {
```

Comment:
Should this ever return a rejected promise which could be handled elsewhere? Just a thought.

Reply:
At least for now I'm putting the error handling in this class. Otherwise both SetEffect and PulseEffect would need to implement their own error handling. Another place that would make sense is PropertyEffect but if a future rule wants to do things with properties (i.e. the Copy effect) I think this Property utility class is the correct place